src/py42/sdk/queries/fileevents/filters/risk_filter.py

```diff
@@ -0,0 +1,232 @@
+        - :attr:`RiskIndicator.CloudDataExposures.SHARED_CORPORATE_ONEDRIVE`
+        - :attr:`RiskIndicator.CloudStorageUploads.AMAZON_DRIVE`
+        - :attr:`RiskIndicator.CloudStorageUploads.BOX`
+        - :attr:`RiskIndicator.CloudStorageUploads.DROPBOX`
+        - :attr:`RiskIndicator.CloudStorageUploads.GOOGLE_DRIVE`
+        - :attr:`RiskIndicator.CloudStorageUploads.ICLOUD`
+        - :attr:`RiskIndicator.CloudStorageUploads.MEGA`
+        - :attr:`RiskIndicator.CloudStorageUploads.ONEDRIVE`
+        - :attr:`RiskIndicator.CloudStorageUploads.ZOHO`
+        - :attr:`RiskIndicator.CodeRepositoryUploads.BITBUCKET`
+        - :attr:`RiskIndicator.CodeRepositoryUploads.GITHUB`
+        - :attr:`RiskIndicator.CodeRepositoryUploads.GITLAB`
+        - :attr:`RiskIndicator.CodeRepositoryUploads.SOURCEFORGE`
+        - :attr:`RiskIndicator.CodeRepositoryUploads.STASH`
+        - :attr:`RiskIndicator.EmailServiceUploads.ONESIXTHREE_DOT_COM`
+        - :attr:`RiskIndicator.EmailServiceUploads.ONETWOSIX_DOT_COM`
+        - :attr:`RiskIndicator.EmailServiceUploads.AOL`
+        - :attr:`RiskIndicator.EmailServiceUploads.COMCAST`
+        - :attr:`RiskIndicator.EmailServiceUploads.GMAIL`
+        - :attr:`RiskIndicator.EmailServiceUploads.ICLOUD`
+        - :attr:`RiskIndicator.EmailServiceUploads.MAIL_DOT_COM`
+        - :attr:`RiskIndicator.EmailServiceUploads.OUTLOOK`
+        - :attr:`RiskIndicator.EmailServiceUploads.PROTONMAIL`
+        - :attr:`RiskIndicator.EmailServiceUploads.QQMAIL`
+        - :attr:`RiskIndicator.EmailServiceUploads.SINA_MAIL`
+        - :attr:`RiskIndicator.EmailServiceUploads.SOHU_MAIl`
+        - :attr:`RiskIndicator.EmailServiceUploads.YAHOO`
+        - :attr:`RiskIndicator.EmailServiceUploads.ZOHO_MAIL`
+        - :attr:`RiskIndicator.RemovableMedia.AIRDROP`
+        - :attr:`RiskIndicator.RemovableMedia.REMOVABLE_MEDIA`
+        - :attr:`RiskIndicator.FileCategories.AUDIO`
+        - :attr:`RiskIndicator.FileCategories.DOCUMENT`
+        - :attr:`RiskIndicator.FileCategories.EXECUTABLE`
+        - :attr:`RiskIndicator.FileCategories.IMAGE`
+        - :attr:`RiskIndicator.FileCategories.PDF`
+        - :attr:`RiskIndicator.FileCategories.PRESENTATION`
+        - :attr:`RiskIndicator.FileCategories.SCRIPT`
+        - :attr:`RiskIndicator.FileCategories.SOURCE_CODE`
+        - :attr:`RiskIndicator.FileCategories.SPREADSHEET`
+        - :attr:`RiskIndicator.FileCategories.VIDEO`
+        - :attr:`RiskIndicator.FileCategories.VIRTUAL_DISK_IMAGE`
+        - :attr:`RiskIndicator.FileCategories.ZIP`
+        - :attr:`RiskIndicator.MessagingServiceUploads.FACEBOOK_MESSENGER`
+        - :attr:`RiskIndicator.MessagingServiceUploads.MICROSOFT_TEAMS`
+        - :attr:`RiskIndicator.MessagingServiceUploads.SLACK`
+        - :attr:`RiskIndicator.MessagingServiceUploads.WHATSAPP`
+        - :attr:`RiskIndicator.Other.OTHER`
+        - :attr:`RiskIndicator.MessagingServiceUploads.UNKNOWN`
```

Comment:
One `MessageServiceUploads` is away from the rest, maybe swap it with the one above?

Reply:
Oh wait, that would should be `:attr:`RiskIndicator.Other.UNKNOWN`